src/components/DateSelectField/index.js

```diff
@@ -147,9 +155,9 @@ const DateSelectField = React.forwardRef((props, ref) => {
         isClearable
         isRequired
         placeholder={label}
-        aria-label={label}
+        aria-label={ariaLabel}
```

Comment:
Probably I'm missing something here, but why not just put `aria-label={${label}-combobox}` directly to select?